### PR TITLE
Add code tabs for _tour/operators

### PR DESCRIPTION
**Pull request by flomebul**

### CODE REVIEW

File: _tour/operators.md [hunk 1]
```diff
@@ -34,27 +47,62 @@ val vector3 = vector1 + vector2
...
+{% endtab %}
+{% endtabs %}
 
 This helps to make the definition of `xor` more readable.
 
 ## Precedence
 When an expression uses multiple operators, the operators are evaluated based on the priority of the first character:
+
+{% tabs operators_6 %}
+{% tab 'Scala 2 and 3' for=operators_6 %}
```
Comment (bishabosha) [inline _tour/operators.md:105]: this example isn't scala code - so I suppose remove the tabs :)

File: _tour/operators.md [hunk 2]
```diff
@@ -67,12 +115,27 @@ When an expression uses multiple operators, the operators are evaluated based on
 |
 (all letters, $, _)
```
Comment (julienrf) [inline _tour/operators.md:116]: Could you also double check https://github.com/scala/docs.scala-lang/issues/2509#issue-1359220737?

File: _tour/operators.md (hunk 1 above)
Comment (flomebul) [reply _tour/operators.md:105]: I agree... I will update this part.

File: _tour/operators.md (hunk 2 above)
Comment (flomebul) [reply _tour/operators.md:113]: Ok. I will change the doc according to the specification.

**Review by julienrf — APPROVED** [review]
Thank you @flomebul!

**Pull request merged**